aspire-orchestrator/README.md

```diff
@@ -1,54 +1,88 @@
-- [dotnet (9 or greater)](https://dotnet.microsoft.com/en-us/download)
-- [Python 3.8 or greater](https://www.python.org/downloads/)
-- [NodeJs v18.12 or greater](https://nodejs.org/en/download/)
+- [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
+- [Docker for Desktop](https://docs.docker.com/get-started/introduction/get-docker-desktop)
+- [dotnet (9 or greater)](https://dotnet.microsoft.com/download)
+- [Python 3.8 or greater](https://www.python.org/downloads)
+- [NodeJs v18.12 or greater](https://nodejs.org/en/download)
 
 ## Run with .NET Aspire
 
-1. Make sure you have accepted the [dotnet trust dev certificate](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-9.0&tabs=visual-studio%2Clinux-sles#trust-the-aspnet-core-https-development-certificate)
-```bash
-dotnet dev-certs https --trust
-```
+1.  Make sure your workstation trusts
+    [ASP.NET Core HTTPS development certificate](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?#trust-the-aspnet-core-https-development-certificate)
 
-2. Clone the repository
+        dotnet dev-certs https --trust
 
-```bash
-make
-cd aspire-orchestrator
-cd Aspire.AppHost
-dotnet run
-```
+2.  Clone the repository
+
+        make
+        cd aspire-orchestrator
+        cd Aspire.AppHost
+        dotnet run
 
 ## Deployment Steps with azd
 
-1. Clone the repository
-2. In the root folder, run
-```bash
-make
-```
-3. [Configure the app registration and configure workbenchapp and workbench service](../docs/CUSTOM_APP_REGISTRATION.md). Set the values in the `aspire-orchestrator/aspire-orchestrator/aspire-orchestrator/aspire-orchestrator/appsettings.json` file
-```json
-{
-    "EntraID": {
-        "ClientId": "<CLIENT_ID>",
-        "Authority": "https://login.microsoftonline.com/<TENANT_ID>"
-    }
-}
-```
-4. Authenticate with Azure Developer CLI
-```bash
-azd login
-```
-5. Create the infrastructure and deploy the application
-```bash
-azd up
-```
-6. Update the App Registration with the correct workbenchapp redirect URI
\ No newline at end of file
+1.  Clone the repository
+
+2.  [Configure the Entra app registration](../docs/CUSTOM_APP_REGISTRATION.md).
+    Copy the Entra **Application Id** and **Tenant Id** into
+    `aspire-orchestrator/Aspire.AppHost/appsettings.json` file.
+
+    If your Entra App allows both organizational and personal accounts
+    use `common` as Tenant Id, e.g. setting
+    `"Authority": "https://login.microsoftonline.com/common"`.
+
+        {
+          "EntraID": {
+            "ClientId": "<CLIENT_ID>",
+            "Authority": "https://login.microsoftonline.com/<TENANT_ID>"
+          }
+        }
+
+3.  In the root folder of the repository, run
+
+        make
+
+4.  Authenticate with Azure Developer CLI
+
+        azd login
+
+5.  Generate the Azure config files required in the next step
+
+        azd init --from-code --environment semanticworkbench
```

Comment:
Are the files this generates excluded by .gitignore?

Reply:
I think so, when I ran these commands I didn't see any changes with `git status`

Reply:
Cool, looks correct as that is the same on my end.